daemon/src/engine/client/cg_msgdef.h

```diff
@@ -719,6 +721,9 @@ using CGameTextInptEvent = IPC::SyncMessage<
 using CGameFocusEventMsg = IPC::SyncMessage<
 	IPC::Message<IPC::Id<VM::QVM, CG_FOCUS_EVENT>, bool>
 >;
+using CGameRecvMessageMsg = IPC::SyncMessage<
+	IPC::Message<IPC::Id<VM::QVM, CG_RECV_MESSAGE>, IPC::SharedMemory, size_t, int>
```

Comment:
It is just that I think we'll need a command buffer in that direction eventually so we could introduce it now.